src/holodeck/agents.py

```diff
@@ -341,14 +377,22 @@ class AndroidAgent(HolodeckAgent):
     There are 18 joints with 3 DOF, 10 with 2 DOF, and 20 with 1 DOF.
 
     Inherits from :class:`HolodeckAgent`."""
+    # constants in Android.h in holodeck-engine
+    __MAX_TORQUE = 20
+    __MIN_TORQUE = -__MAX_TORQUE
 
     agent_type = "Android"
 
     @property
     def control_schemes(self):
-        return [("[Raw Bone Torques] * 94", ContinuousActionSpace([94])),
+        direct_min = [self.__MIN_TORQUE for _ in range(94)]
```

Comment:
I think 94 should be a const variable. You use it many times and it may change in the future. It would also help readability

Reply:
Done.

Reply:
I sure hope that number never changes haha. It's magic is found throughout the codebase